plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSessionProperties.java

```diff
@@ -55,6 +55,7 @@
     private static final String DYNAMIC_FILTERING_WAIT_TIMEOUT = "dynamic_filtering_wait_timeout";
     private static final String TABLE_STATISTICS_ENABLED = "statistics_enabled";
     private static final String EXTENDED_STATISTICS_ENABLED = "extended_statistics_enabled";
+    private static final String TARGET_MAX_FILE_SIZE = "target_max_file_size";
```

Comment:
nit: I would move to around between `PARQUET_WRITER_PAGE_SIZE` and `PARQUET_OPTIMIZED_WRITER_ENABLED` since this property is for writer. (constructor, getter & config test is also)